### PR TITLE
BookController implemented

### PR DESCRIPTION
**Pull request by p1q**

### CODE REVIEW

File: src/main/java/library/spring/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@RequestMapping("/")
+public class BookController {
+    private static final Logger LOGGER = Logger.getLogger(BookController.class);
+
+    @Autowired
+    private BookService bookService;
+    @Autowired
+    private RentService rentService;
+
+    @GetMapping("/show-all-books")
```
Comment (Sofasmile) [inline src/main/java/library/spring/controller/BookController.java]:
```suggestion
    @GetMapping("/all")
```

File: src/main/java/library/spring/controller/BookController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    public String getAllBooks(ModelMap model) {
+        model.put("books", bookService.getAllBooks());
+        return "show-all-books";
+    }
+
+    @GetMapping("/show-book-info")
```
Comment (Sofasmile) [inline src/main/java/library/spring/controller/BookController.java]:
```suggestion
    @GetMapping("/info")
```

File: src/main/java/library/spring/controller/BookController.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@RequestMapping("/")
```
Comment (Sofasmile) [inline src/main/java/library/spring/controller/BookController.java]:
```suggestion
@RequestMapping("/book")
```

File: src/main/java/library/spring/controller/BookController.java (hunk 1 above)
Comment (p1q) [reply src/main/java/library/spring/controller/BookController.java]: Done

File: src/main/java/library/spring/controller/BookController.java (hunk 2 above)
Comment (p1q) [reply src/main/java/library/spring/controller/BookController.java]: Done

File: src/main/java/library/spring/controller/BookController.java (hunk 3 above)
Comment (p1q) [reply src/main/java/library/spring/controller/BookController.java]: Done

**Pull request merged**